workshops/geoserver/intro/slides/slides.rst

```diff
@@ -60,50 +60,43 @@ Being a community-driven project, GeoServer is developed, tested, and supported
 Section 1: Installing
 =====================
 
-GeoServer is a core component of **OpenGeo Suite**.
+Workshop uses your choice of:
 
-.. image:: ../doc/source/install/img/ecosystem.png
-   :width: 50%
+* Boundless Suite GeoServer Distribution
+* OSGeo Community GeoServer Distribution
 
 Presenter notes
 ---------------
 
-In this section, we will install GeoServer. For the purposes of this workshop, we will be using the OpenGeo Suite—of which GeoServer is a primary component—in order to facilitate setup and configuration.
+In this section, we will install GeoServer.
 
-The OpenGeo Suite is free and open source, and is available for download from Boundless.
+GeoServer is a popular component, this workshop can use your choice of Boundless Suite or OSGeo Community downloads.
 
-OpenGeo Suite is a complete web-based geospatial software stack. In this package, the applications contained are:
+Boundless Suite is available in a standalone virtual machine.
 
-PostGIS - A spatially enabled object-relational database.
-GeoServer - A software server for loading and sharing geospatial data.
-GeoWebCache - A tile cache server that accelerates the serving of maps (built into GeoServer).
-OpenLayers - A browser-based mapping framework
+OSGeo Community downloads will install directly onto your machine; and will require several optional downloads for this workshop.
```

Comment:
Suggest: "The version of GeoServer community distribution will install directly onto your machine"